src/docker-utils.coffee

```diff
@@ -101,27 +101,30 @@ do ->
 		Promise.using readLockImages(), ->
 			dockerProgress.pull(image, onProgress)
 
-	supervisorTag = config.supervisorImage
-	if !/:/g.test(supervisorTag)
-		# If there is no tag then mark it as latest
-		supervisorTag += ':latest'
+	normalizeRepoTag = (image) ->
```

Comment:
It does, but I'd rather have it here for now (`buildRepoTag` is used below in other places) and split it out later, so that we can fix this bug quickly.